wiki-docs/docs/interfaces/validators/root-validator-set.md

```diff
@@ -0,0 +1,73 @@
+The `IRootValidatorSet` interface provides functionality for managing the validator set on the rootchain. It allows adding new validators and querying validator information. This user guide will explain how to interact with the functions provided by the `IRootValidatorSet` interface.
```

Comment:
Remove this file. This interface and contract is no longer used.

Reply:
done